Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apt-get install -y $(grep -vE "^\s*#" apt.txt  | tr "\n" " ")
 #############################################
 
 # Add repo needed for postgres 9.6 and install it
-RUN apt-get update && apt-get install libpq-dev postgresql-client-9.6 -y
```

Comment:
I don't think we should be installing `sudo` in case down the line we start using this dockerfile down the road, Im also not clear on why you need it, could you explain?